cmd/plugins/topology-aware/policy/resources.go

```diff
@@ -1105,6 +1138,33 @@ func (cs *supply) GetScore(req Request) Score {
 
 		// calculate fractional capacity
 		score.shared -= part
+
+		ecores := cs.GetNode().System().CoreKindCPUs(sysfs.EfficientCore)
+		lpCPUs := ecores
+		if ecores.Size() == 0 {
+			lpCPUs = cs.GetNode().Policy().cpuAllocator.GetCPUPriorities()[lowPrio]
+		}
+		lpCPUs = lpCPUs.Intersection(cs.SharableCPUs())
+		lpCnt := lpCPUs.Size()
+
+		score.prio[lowPrio] = lpCnt*1000 - (1000*full + part)
+
+		pcores := cs.GetNode().System().CoreKindCPUs(sysfs.PerformanceCore)
+		hpCPUs := pcores
+		if pcores.Size() == 0 {
+			hpCPUs = cs.GetNode().Policy().cpuAllocator.GetCPUPriorities()[highPrio]
+		}
+		hpCPUs = hpCPUs.Intersection(cs.SharableCPUs())
+		hpCnt := hpCPUs.Size()
+		score.prio[highPrio] = hpCnt*1000 - (1000*full + part)
+
+		npCPUs := pcores
+		if pcores.Size() == 0 {
+			npCPUs = cs.GetNode().Policy().cpuAllocator.GetCPUPriorities()[highPrio]
+		}
+		npCPUs = npCPUs.Intersection(cs.SharableCPUs())
+		npCnt := npCPUs.Size()
+		score.prio[normalPrio] = npCnt*1000 - (1000*full + part)
```

Comment:
Looks like we get the exact same calculation and result for hpCPUs and npCPUs 🤨 Is that the intention (and if so, we don't need to calculate it twice)

Reply:
Updated to use normal priority instead of trying P-cores for the normal prio request.